src/views/Pools/components/StakeModal.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Modal, Text, Flex, Image, Button, Slider, RefreshIcon } from '@pancakeswap-libs/uikit'
+import useI18n from 'hooks/useI18n'
+import { BASE_EXCHANGE_URL } from 'config'
+import { useSousStake } from 'hooks/useStake'
+import { useSousUnstake } from 'hooks/useUnstake'
+import useTheme from 'hooks/useTheme'
+import BigNumber from 'bignumber.js'
+import { getFullDisplayBalance } from 'utils/formatBalance'
+import { Address } from 'config/constants/types'
+
+import BalanceInput from './BalanceInput'
+
+interface StakeModalProps {
+  sousId: number
+  isBnbPool: boolean
+  stakingTokenDecimals?: number
+  stakingTokenName: string
+  stakingTokenAddress?: Address
+  max: BigNumber
+  isStaking?: boolean
+  onDismiss?: () => void
+}
+
+const StyledButton = styled(Button)`
+  font-size: 12px;
+  margin: 0 2px;
+  flex-grow: 1;
+  padding: 4px 16px;
+  height: 20px;
+  max-width: 57px;
```

Comment:
where this maxwidth come from ?

Reply:
To make button look same as one in the Figma, I added it.